src/Controller/Controller.php

```diff
@@ -565,7 +559,8 @@ public function setAction($action)
     public function render($view = null, $layout = null)
     {
         if (!empty($this->request->params['bare'])) {
-            $this->getView()->autoLayout = false;
+            $this->View = $this->getView();
+            $this->_view->autoLayout = false;
```

Comment:
No it's not. `$this->View` is set only for backwards compatibility. I added new protected property `ViewVarsTrait::$_view` for storing view instance.